Content.Server/Chat/ChatSystem.cs

```diff
@@ -330,6 +331,31 @@ private void SendInVoiceRange(ChatChannel channel, string message, string messag
         _chatManager.ChatMessageToMany(channel, message, messageWrap, source, hideChat, sessions.Select(s => s.ConnectedClient).ToList());
     }
 
+    /// <summary>
+    ///     Input a string, outputs the correct tone that you can get with Loc.GetString
+    /// </summary>
+
+    private string GetToneBySuffix(string s) {
+        var toneSuffixes = new Dictionary<string,string>() {
+            {"?!", "chat-manager-entity-exclaims-wrap-message"},
+            {"!?", "chat-manager-entity-exclaims-wrap-message"},
+            {"?", "chat-manager-entity-ask-wrap-message"},
+            {"!!!", "chat-manager-entity-scream-wrap-message"},
+            {"!!", "chat-manager-entity-shout-wrap-message"},
+            {"!", "chat-manager-entity-yell-wrap-message"},
+        };
```

Comment:
Should be a prototype.

Reply:
should i ditch the FTL file in favor of putting the messages in the prototype or just continue with what im doing and add prototypes

Reply:
this doesn't need to be a prototype, its short enough

Reply:
it -should- just be a private member field on `ChatSystem` though